lisa/operating_system.py

```diff
@@ -794,6 +800,13 @@ def _install_packages(
         install_result = self._node.execute(
             command, shell=True, sudo=True, timeout=timeout
         )
+        if self._debian_package_install_unauthenticated_regex.search(
+            install_result.stdout
+        ):
+            command += " --allow-unauthenticated"
```

Comment:
Can we add it as parameter to the function?

Reply:
it was already a parameter signed, signed = False, then `--allow-unauthenticated` will be appended, but from current test results, there are some packages from the old distro (debian) need specify --allow-unauthenticated, that means we can't know when to use this beforehand.